import "mapreduce/yt/interface/protos/extension.proto";

message TDaemonLogRow {
    optional string SpravkaIp = 1       [(NYT.column_name) = "spravka_ip"];
    optional string ProcessorHost = 2   [(NYT.column_name) = "processor_host"];
    optional string CacherHost = 3      [(NYT.column_name) = "cacher_host"];
}
